gunicorn.py

```diff
@@ -1,9 +1,11 @@
 import multiprocessing
+import os
 
 from robotoff import models
 
 bind = ":5500"
-workers = multiprocessing.cpu_count() * 2 + 1
+# we have a trade-off with memory vs cpu numbers
+workers = os.environ.get("GUNICORN_NUM_WORKERS", 4)
```

Comment:
casting to an int as env variables are loaded as strings.

```suggestion
workers = int(os.environ.get("GUNICORN_NUM_WORKERS", 4))
```

Reply:
oops ! right !